mmv1/README.md

```diff
@@ -0,0 +1,38 @@
+# MMV1
+
+## Overview
+
+Mmv1 is a code generator that implements the Terraform Google Provider (TGP) resources from ruby scripts.
```

Comment:
WDYT of something like "MMv1 is a Ruby-based code generator that ... from YAML specification files. Contributions typically require editing YAML files and small snippets of Go code."? Other than by folks on our team, Ruby contributions are pretty rare.

Reply:
Totally make sense! I'll change it to "MMv1 is a Ruby-based code generator that implements Terraform Provider Google (TPG) resources from YAML specification files." or should I also mention TPGB? But I was thinking TPG may be a general term for the google providers.

Reply:
Yeah- TPG generally means both.